### ISSUE TITLE
Обновление плейлистов

### ISSUE THREAD

**Issue by s0d2** [delletenebre/Yamus#8]
Часто наблюдаю, что плейлист дня и прочие подборки, которые в сервисе обновляются ежедневно, в клиенте Ямуз для телефона не первой свежести. Может пройти и два и три дня с момента обновления.

**Comment by dojekon** [comment]
Да, есть такой баг. Но нашёл решение. Нужно открыть приложение ЯМуз на телефоне, полистать вкладки, потыкать кнопки :) После этого обновляется

**Comment by delletenebre** [comment]
На яндекс диск выложил мод приложения ЯндексМузыка с активированной функцией AndroidAuto. Как оказалось, данный режим активируется одной строчкой :)

**Comment by h4tr3d** [comment]
@delletenebre немного не в тему, но вопрос: а как в приложении из результатов поиска добавить playlist в избранное?
